app/models/ai_persona.rb

```diff
@@ -0,0 +1,103 @@
+# frozen_string_literal: true
+
+class AiPersona < ActiveRecord::Base
+  class MultisiteHash
+    def initialize(id)
+      @hash = Hash.new({})
+      @id = id
+
+      MessageBus.subscribe(channel_name) { |message| @hash[message.data] = {} }
```

Comment:
You can get the db from `message.site_id`, you don't need to include it in the message.

Reply:
that is a bit risky, it would depend on an assumption here (which happens to be correct, but who knows)